.gitignore

```diff
@@ -2,3 +2,5 @@ pkg/*
 **/.DS_Store
 .rspec
 gemfiles
+.ruby-version
```

Comment:
It should, quickly checking our circle file I see we are only testing against `2.2.4`.  Let's set it at that and I'll add a task to update our CI.  I think I tested it manually against `2.3.0` when it was first released in December and it was failing.